OpenRA.Mods.Common/Traits/BotModules/SquadManagerBotModule.cs

```diff
@@ -279,7 +279,7 @@ void TryToRushAttack(IBot bot)
 			// TODO: This should use common names & ExcludeFromSquads instead of hardcoding TraitInfo checks
 			var ownUnits = activeUnits
 				.Where(unit => unit.IsIdle && unit.Info.HasTraitInfo<AttackBaseInfo>()
-					&& !unit.Info.HasTraitInfo<AircraftInfo>() && !unit.Info.HasTraitInfo<HarvesterInfo>()).ToList();
+                    && !unit.Info.HasTraitInfo<AircraftInfo>() && !Info.NavalUnitsTypes.Contains(unit.Info.Name) && !unit.Info.HasTraitInfo<HarvesterInfo>()).ToList();
```

Comment:
Please keep this as tabs instead of spaces.

Reply:
oh weird. I always use tabs. Not sure why it converted.

Reply:
Strange its showing as tabs on my local file.

Reply:
Ok fixed. Also added in one more spot to check for naval.